aip/auth/4114.md

```diff
@@ -114,13 +127,16 @@ available. The default value of this environment variable will be "auto".
 **GOOGLE_API_USE_CLIENT_CERTIFICATE**: If **"true"**, device certificate
 authentication will be supported as described in the general guidance. If
 **"false"**, the device certificate **must** not be used, even if specified by
-the user. For now, the default value will be "false", since mTLS support is not
-yet fully adopted by all services. Users who wish to enable DCA feature **must**
-explicitly set this environment variable to "true". In the future, the default
-value will be "true' to allow a more secure connection to be established
-whenever possible.
+the user. The default value **should** be "true" as of May 3, 2024. Users who
```

Comment:
```suggestion
the user. The default value is "true" as of May 3, 2024. Users who
```

Is it in fact true everywhere? If so, just make it a statement. By using a requirement word like "should" we leave it open to ambiguity i.e. we think it should be but maybe it isn't. Notice the original text was matter-of-fact: "the default value will be 'false'"

Reply:
The wording "should" is intentional here, as not all supported languages have migrated to true by default (only Golang client has made the switch so far). Java and Python still needs to flip the switch, but we have no ETA on that as of now.